client/packages/electron/src/electron.ts

```diff
@@ -210,6 +215,27 @@ const start = (): void => {
     },
   });
 
+  // Not using i18next here, as that needs more of the app to be initialised
+  // than we have available at this point. We also don't need all the translations,
+  // just the desktop ones, so we can import them directly.
+
+  const appLocale = app.getLocale();
```

Comment:
As someone who doesn't use a different language on my OS I was wondering if this was the source of truth, or using the user selected language in app - though that appears to be from local storage and maybe over ruled by browser language if using in browser

![image](https://github.com/user-attachments/assets/f6f0d6ef-2fe5-4968-bd5f-e809da94ced4)

So can get a bit of misalignment anyways. Not a huge deal I think and can be changed later